.github/lock.yml

```diff
@@ -0,0 +1,13 @@
+# Configuration for lock-threads - https://github.com/dessant/lock-threads
+
+# Number of days of inactivity before a closed issue or pull request is locked
+daysUntilLock: 180
+# Comment to post before locking. Set to `false` to disable
+lockComment: >
+  This issue is now locked since it had enough time for everyone to give further
```

Comment:
> This issue is now locked since it had enough time for everyone to give further feedback on after it was closed.

I think at the least this should have the "after it was closed" part removed, since the timeout is based on inactivity, not the close date.

> This issue is now locked as it has had enough time for everyone to give further feedback on it.

Or alternatively, with more finality to it:

> This issue is now locked as the time for any further feedback has passed.

Reply:
I like the the conciseness of the last form 👍

Reply:
The bot only locks closed issues. I wanted to have the keyword `closed` in there to communicate that only closed issues are locked. But I agree that the current wording might be misleading. I don't get any immediate ideas on how to change this to include both things :thought_balloon:

Clearly communicating how and why the bot acts is something I learned is important when rewording the stale message after feedback. If we remove closed there is someone that will believe we started locking all issues and PRs.

Reply:
Maybe

> This issue has been automatically locked since there has not been any recent activity after it was closed.

Reply:
> The bot only locks closed issues. I wanted to have the keyword closed in there to communicate that only closed issues are locked.

This message would only be shown on closed issues, but making it explicit doesn't hurt, especially since this would likely be the last comment on these threads.

Your new message works for me!